webui/react/.stylelintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   extends: [ 'stylelint-config-standard' ],
-  plugins: [ 'stylelint-order' ],
+  plugins: [ 'stylelint-order', 'stylelint-scss' ],
   rules: {
+    'at-rule-no-unknown': null,
```

Comment:
nice! what at-rule did you encounter to add this? just curious

Reply:
@extend. I added the one from scss and that recognized it